stratum/hal/lib/phal/phal.cc

```diff
@@ -65,16 +62,18 @@ ::util::Status Phal::PushChassisConfig(const ChassisConfig& config) {
     std::vector<std::unique_ptr<SwitchConfiguratorInterface>> configurators;
 
     // Set up ONLP
-#if defined(WITH_ONLP)
     {
       auto* onlp_wrapper = onlp::OnlpWrapper::CreateSingleton();
       auto* onlp_phal = onlp::OnlpPhal::CreateSingleton(onlp_wrapper);
-      phal_interfaces_.push_back(onlp_phal);
-      ASSIGN_OR_RETURN(auto configurator, onlp::OnlpSwitchConfigurator::Make(
-                                              onlp_phal, onlp_wrapper));
-      configurators.push_back(std::move(configurator));
+      if (onlp_phal) {
+        phal_interfaces_.push_back(onlp_phal);
+        ASSIGN_OR_RETURN(auto configurator, onlp::OnlpSwitchConfigurator::Make(
+                                                onlp_phal, onlp_wrapper));
+        configurators.push_back(std::move(configurator));
+      } else {
+        LOG(INFO) << "ONLP disabled.";
```

Comment:
```suggestion
        LOG(WARNING) << "ONLP disabled.";
```

Reply:
ONLP library could not be loaded. ONLP is disabled.

Reply:
max to pick the right level